srcpkgs/freerdp3/template

```diff
@@ -50,8 +51,9 @@ if [ "$build_option_webview" -a -z "$build_option_sdl2" ]; then
 fi
 
 case "$XBPS_TARGET_MACHINE" in
-	i686*|x86_64*) configure_args+=" -DWITH_SSE2=ON"  ;;
-	armv5tel*)     configure_args+=" -DWITH_NEON=OFF" ;;
+	i686*|x86_64*) configure_args+=" -DWITH_SSE2=ON"   ;;
+	armv5tel*)     configure_args+=" -DWITH_NEON=OFF"  ;;
```

Comment:
i don't think we can assume neon support on all arm*

Reply:
switch it off then for all arm*?

Reply:
yes